testsupport/init.go

```diff
@@ -138,6 +138,11 @@ func WaitForDeployments(t *testing.T) wait.Awaitilities {
 		err = initHostAwait.WaitUntilBaseNSTemplateTierIsUpdated()
 		require.NoError(t, err)
 
+		// check that the default user tier exists and is updated to the current version, an outdated version is applied from deploy/e2e-tests/usertier-base.yaml as
+		// part of the e2e test setup make target for the purpose of verifying user tiers are updated on startup of the host operator
+		err = initHostAwait.WaitUntilBaseUserTierIsUpdated()
+		require.NoError(t, err)
+
```

Comment:
the e2e tests are failing because this code is executed in the "migration setup part" - which means before the UserTier CRD is created.

Reply:
yeah, I'm aware of the failure. Do you have any suggestion on how to fix this? I was thinking of just commenting this out for the initial PR and then uncommenting it in a separate PR. WDYT?

Reply:
yup, that would work :+1: 
another option would be:
1. comment it out
2. let the host-operator PR pass
3. merge host-operator PR
4. uncomment it and let it pass as well
5. merge this PR

I know that ideally we should follow the approach when we merge both PRs together, but there is no breaking change, so it's not a big deal. 
I'll leave it up to you - either way is fine.

Reply:
cool, I'll use your way since it'll save me the trouble of having to create a separate PR 🙂

Reply:
hmm, looks like the branch pairing fails if the host operator PR is merged so let me create a separate PR to merge these tests

Reply:
new PR: https://github.com/codeready-toolchain/toolchain-e2e/pull/548

Reply:
ah, right - I totally forgot about this. Sorry for causing that trouble :-)